test/e2e/windows/host_process.go

```diff
@@ -109,9 +116,10 @@ var _ = SIGDescribe("[Feature:WindowsHostProcessContainers] [MinimumKubeletVersi
 				HostNetwork: true,
 				Containers: []v1.Container{
 					{
-						Image:   image.GetE2EImage(),
-						Name:    "computer-name-test",
-						Command: []string{"cmd.exe", "/K", "IF", "NOT", "%COMPUTERNAME%", "==", targetNode.Name, "(", "exit", "-1", ")"},
+						Image: image.GetE2EImage(),
+						Name:  "computer-name-test",
+						// The /I argument makes the string comparison case insensitive.
+						Command: []string{"cmd.exe", "/K", "IF", "/I", "NOT", "%COMPUTERNAME%", "==", computerName, "(", "exit", "-1", ")"},
```

Comment:
@claudiubelu ...  wait... shouldnt this be

```
if $COMPUTERNAME%.contains(computerName) ;
   pass
else
   fail
```
?

Reply:
Just as a sample:

```
C:\Users\Administrator\workdir\kubernetes>echo %COMPUTERNAME:~0,15%
WIN-AE7D7USH1E2

C:\Users\Administrator\workdir\kubernetes>echo %COMPUTERNAME:~0,5%
WIN-A
```